azure-proto-core/ArmClient.cs

```diff
@@ -1,18 +1,18 @@
-﻿using Azure;
-using Azure.Core;
-using Azure.Identity;
-using Azure.ResourceManager.Resources;
-using Azure.ResourceManager.Resources.Models;
-using azure_proto_core.Adapters;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Runtime.CompilerServices;
-using System.Threading;
-using System.Threading.Tasks;
-
-namespace azure_proto_core
+﻿namespace azure_proto_core
 {
+    using System;
```

Comment:
We will need to change this to outside. Let me know if the ruleset was incorrectly set for this one.